src/contracts/middleware/Middleware.sol

```diff
@@ -516,30 +351,14 @@ contract Middleware is SimpleKeyRegistry32, Ownable, IMiddleware {
     //                                      VIEW FUNCTIONS
     // **************************************************************************************************
 
-    /**
-     * @inheritdoc IMiddleware
-     */
+    // /**
+    //  * @inheritdoc IMiddleware
+    //  */
```

Comment:
I've seen many of these. I'm guessing either `IMiddleware` changed or we stopped inheriting from it. Shall we update/create a new interface?